components/desktop/adwaita-icon-theme/Makefile

```diff
@@ -25,7 +25,7 @@ COMPONENT_SUMMARY=		GNOME Icon Theme
 COMPONENT_SRC=			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=		$(COMPONENT_SRC).tar.xz
 COMPONENT_ARCHIVE_HASH=	sha256:6683a1aaf2430ccd9ea638dd4bfe1002bc92b412050c3dba20e480f979faaf97
-COMPONENT_ARCHIVE_URL=	https://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/$(COMPONENT_MJR_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_URL=	https://download.gnome.org/sources/$(COMPONENT_NAME)/$(COMPONENT_MJR_VERSION)/$(COMPONENT_ARCHIVE)
 COMPONENT_FMRI=			gnome/theme/adwaita-icon-theme
```

Comment:
Re-releasing a package needs a change in the COMPONENT_REVISION (that is either increment it if already exisiting or set it to 1). It's technically not needed because I have incremented the RELEASE_MAJOR recently but alarcher prefers to have the COMPONENT_REVISION incremented still.

Reply:
This change does not need re-release of any package.

Reply:
Any change in a Makefile will trigger a re-release by our jenkins.

Reply:
That should not cause any problem, I assume.

Reply:
It's not causing problems because I changed the RELEASE_MAJOR but you should keep that in mind for future PR's.

Reply:
It wouldn't cause any problem even without RELEASE_MAJOR change.

So, should I do any change for this PR?

Reply:
Sure.